A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/A32NX_BaseAirliners.js

```diff
@@ -188,7 +188,7 @@ var Airliners;
             }
         }
         onUpdate() {
-            const _deltaTime = this.getDeltaTime();
+            var _deltaTime = this.getDeltaTime();
```

Comment:
No reason to change this one.

```suggestion
            const _deltaTime = this.getDeltaTime();
```